docs/config/session.rst

```diff
@@ -212,6 +212,13 @@ releases.
    The optimizer setting is for advanced use only and can significantly impact
    the performance behavior of the queries.
 
+**optimizer_eliminate_cross_join**
```

Comment:
Vale thinks it found a mistake here. Is there a canonical way to suppress it?

Reply:
Maybe wrapping with `` would help?

Reply:
If this doesn't help there is
```
..vale off
<context-to-suppress>
..vale on
```